tests/unit_tests/Models/SiteTest.php

```diff
@@ -540,6 +540,7 @@ public function testSerialize()
             'tags' => implode(',', $tags),
             'max_num_cdes' => 0,
             'last_frozen_at' => '1682641540',
+            'region' => null,
```

Comment:
Here is an existing unit test that is exercised by this modification. Do you want any additional coverage by any unit tests?